catboost/python-package/catboost/core.py

```diff
@@ -1616,6 +1616,15 @@ def get_feature_importance(self, data=None, type=EFstrType.FeatureImportance, pr
 
         fstr_type : string, deprecated, use type instead
 
+        shap_mode : string, optional (default="Auto")
+            used only for ShapValues type
+            Possible values:
+                - "Auto"
+                    Use direct SHAP Values calculation only if data size is smaller than average leaves number
```

Comment:
The comment should be clear and informative for python library users. For example, you can write complexity here and an advice when to use which one.